mongodb-odbc-driver/bin/build-mongo-odbc.sh

```diff
@@ -78,10 +78,17 @@
     if [ "$PLATFORM_NAME" = "windows" ]; then
         cp "$BUILD_DIR"/lib/Release/mdbodbc{a,S,w}.{dll,lib} "$DRIVERS_DIR"
         cp "$BUILD_DIR"/bin/Release/myodbc-installer.exe "$DRIVERS_DIR"/installer.exe
-		#also copy openssl
-		cp "$OPENSSL_PATH"/*.dll "$DRIVERS_DIR"
+        #also copy openssl
+        cp "$OPENSSL_PATH"/*.dll "$DRIVERS_DIR"
     else
         cp "$BUILD_DIR"/lib/libmdbodbc{a,w}.so "$DRIVERS_DIR"
+        # if this is mac, we need to update the openssl library rpaths
```

Comment:
This is the important thing, this allows it to link against openssl in the same directory as the driver library ('@loader_path' refers to the location of the driver library, current directory is where the library is called from).

Reply:
This is a helpful explanation! Could you include it as a source-code comment as well?

Reply:
Which source code?

Reply:
Added it in as a suggestion! Sorry, could have make that more clear

Reply:
oh, I get you now!